Makefile

```diff
@@ -433,6 +434,7 @@ deep-copy: codegen-tools ## Deep copy
 	@$(SHELL) $(CURDIR)/agent/proxycfg/deep-copy.sh
 	@$(SHELL) $(CURDIR)/agent/consul/state/deep-copy.sh
 	@$(SHELL) $(CURDIR)/agent/config/deep-copy.sh
+	copywrite headers
```

Comment:
Do we need to limit this to just `*.deepcopy.go` so that other files aren't caught by the check? Maybe there is cause to refactor this to check to all files unless we're already doing that somewhere else.

Reply:
We are not doing this to all the files right now. My thinking is that we should, as I already found out few files which don't have the copyright header.
That said, maybe it would be better to make it independent from deep-copy 🤔